presets/ragengine/inference/inference.py

```diff
@@ -39,28 +40,30 @@ def stream_complete(self, prompt: str, **kwargs: Any) -> CompletionResponseGen:
         pass
 
     @llm_completion_callback()
-    def complete(self, prompt: str, formatted: bool, **kwargs) -> CompletionResponse:
+    def complete(self, prompt: str, formatted: bool = False, **kwargs: Any) -> CompletionResponse:
+        # since acomplete is async, we can run it in a blocking way here
+        return asyncio.run(self.acomplete(prompt, formatted=formatted, **kwargs))
+
+    @llm_completion_callback()
+    async def acomplete(self, prompt: str, formatted: bool = False, **kwargs: Any) -> CompletionResponse:
```

Comment:
acomplete -> async_complete

Reply:
this class is overriden from the CustomLLM interface class - so i cant change its name